src/Uno.UI/UI/Xaml/Controls/ListViewBase/VirtualizingPanelLayout.Android.cs

```diff
@@ -1435,6 +1435,21 @@ private void ScrapLayout(RecyclerView.Recycler recycler, int availableBreadth)
 			_areHeaderAndFooterCreated = false;
 		}
 
+		// If there are no groups, this probably means that the source is grouped and Header or Footer are pushing all items completely out of view.
+		int? GetDynamicStartFromHeader()
+		{
+			if (HeaderViewCount > 0)
```

Comment:
if there's a header, having a footer is irrelevant?

Reply:
If there's a Header, and a Footer, and nothing else, then the end of the Header == start of the Footer.